actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -72,6 +73,7 @@ def rack_response
       def write(data)
         @stream.write(data)
       rescue => e
+        @logger.error "An #{e.class} error occurred while writing to the ActionCable socket: #{e.message}"
```

Comment:
Can we change the error message to `"#{e.class} occurred while trying to write to the Action Cable socket: #{e.message}`? That we we don't have to worry about a/an